Movies-MVC/Movies-MVC/Models/NetworkInfo.swift

```diff
@@ -0,0 +1,26 @@
+// NetworkInfo.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Модель списка фильмов
+struct Results: Codable {
+    let results: [Movie]
```

Comment:
```suggestion
    let movies: [Movie]
```
массив и тип должны одинаково называться